unpub/lib/src/static/style.dart.css.dart

```diff
@@ -0,0 +1,2 @@
hite-space:nowrap}@media(min-width: 641px){.packages-scores .packages-score-label{font-size:12x;letter-spacing:.2px}}.detail-info-box .packages-scores .packages-score-label{text-align:left}.search-bar{display:flex;align-items:center;border-radius:3px;max-width:710px;background:transparent;position:relative;margin:0 auto;padding:13px 0}.home-banner .search-bar{max-width:880px}.search-bar>.input{font-size:16px;padding:15px 24px 14px 48px;border:none;border-radius:24px;width:150px;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;background:#35404d;color:#fff}.search-bar>.input::placeholder{color:#888}@media(min-width: 641px){.home-banner .search-bar>.input{font-size:24px;padding:20px 35px 20px 65px;border-radius:35px}}.search-bar>.input:focus{outline:none}.search-bar>.icon{width:45px;height:45px;background-color:#0175c2;background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAABGdBTUEAALGPC/xhBQAAARdJREFUOBHNkz1uwkAQhTGkhCuELlQUiAtwilzHtU8BVLkHF4iokbJJkS41EiTS5nvL2FrQGMsdIz3N7ntvZn+8HgyyiDEuwBv4BifLmi8yW/sQ4xMowRl4Ib4Eo/YuKGYixT9QgSUYW9ZcvKJsbYSo42hFmVeeUbzp8vnHRNAdKCqvSc1JTy78NXeVEXWxiuWVcDORnlz4b6TLFFFfRzF2DUaiT5IrxqPnG0L+mDDzDBn3YuNDxjVDNdrZ7LVh/UGtv7sy2+3z1X7xz91GIhFLoOh6R+vWJtao62VrgW1aKsZP8rSrYf6vHSnYgw1IxyEHoAjgfrN7K6kYBKAI4DGbfbCzZ72j3lEUxRdFKxB6F3sF2okg7R/6f6gee6OuegAAAABJRU5ErkJggg==");background-repeat:no-repeat;background-position:50% 50%;background-color:transparent;opacity:.4;position:absolute;left:10px;transform:scale(0.72)}@media(min-width: 641px){.home-banner .search-bar>.icon{left:20px;transform:scale(1)}}.search-filters-btn-wrapper{display:none}@media(max-width: 640px){.search-filters-btn-wrapper{display:block}}.search-filters-btn-wrapper .search-filters-btn{cursor:pointer;display:block;width:40px;height:40px;margin-left:8px}.search-filters-btn-wrapper .search-filters-btn-inactive{display:block}.search-filters-btn-wrapper .search-filters-btn-active{display:none}.search-filters-btn-wrapper.-active .search-filters-btn-inactive{display:none}.search-filters-btn-wrapper.-active .search-filters-btn-active{display:block}.search-bar-details{position:relative;max-width:650px}.search-bar-options{font-size:14px}@media(min-width: 600px){.search-bar-options{position:absolute;top:16px;right:0px}}.package-tag{background:#eef9fe;text-transform:uppercase;display:inline-block;font-size:12px;margin:0px 0 4px 4px;color:#2465df;padding:7px 10px}.package-tag.missing{background:#f0f0f0;color:#555}.package-tag.unidentified,.package-tag.unlisted{background:#fff0f0;color:#555}.package-tag.legacy,.package-tag.discontinued{background:#c0392b;color:#f8f8f8}.-pub-tag-badge{font-size:12px;display:inline-block;margin-right:15px;background:#e7f8ff}.-pub-tag-badge>.tag-badge-main,.-pub-tag-badge>.tag-badge-sub{display:inline-block;text-transform:uppercase;padding:4px 8px;color:#1967d2}.-pub-tag-badge>.tag-badge-main{color:#1967d2;border-right:1px solid rgba(25,103,210,.5);font-weight:500}/*# sourceMappingURL=style.css.map */""";
```

Comment:
maybe format the css to make it more readable? but i leave it up to the maintainer to decide..